isofit/utils/apply_oe.py

```diff
@@ -276,6 +277,9 @@ def apply_oe(args):
     elif args.sensor == "prisma":
         # parse flightline ID (PRISMA assumptions)
         dt = datetime.strptime(paths.fid, "%Y%m%d%H%M%S")
+    elif args.sensor == "gao":
+        # parse flightline ID (GAO/CAO assumptions)
+        dt = datetime.strptime(paths.fid[3:-5], '%Y%m%dt%H%M%S')
```

Comment:
Please replace the single quotes with double quotes in order to pass the Black linting. In order to ensure linting is automatically applied in the future, you can run a `pre-commit install` in the root directory of the project.

Reply:
Pushed the double quote changes. Thanks!